save-cosv/src/main/kotlin/com/saveourtool/save/cosv/controllers/RawCosvFileController.kt

```diff
@@ -269,7 +270,9 @@ class RawCosvFileController(
         authentication: Authentication,
     ): Mono<StringResponse> = rawCosvFileStorage.listByOrganizationAndUser(organizationName, authentication.name)
         .map { files ->
-            files.map { it.requiredId() }
+            files
+                .filter { !it.isZipArchive() && it.status == RawCosvFileStatus.UPLOADED }
```

Comment:
May be even create `it.isJsonFile()`?

Reply:
will create a new issue for it. I think we should also skip non json and non zip files on backend side

Reply:
created #2802